source/api/discovery/0.5/index.md

```diff
@@ -851,19 +865,19 @@ Given an array (`collections`) of collection URIs as input,
 
 ### 4.1. Activities for Access-Restricted Content
 
-Activities _MAY_ be published about content that has access restrictions. Clients _MUST NOT_ assume that they will be able to access every resource that is the object of an Activity, and _MUST NOT_ assume that if they cannot access it, then it has been deleted and therefore remove it from their indexes.  For example, the content might be protected by an authentication system that is denying access, or there might simply be a temporary network outage preventing the content from being retrieved. An end user might be able to provide the right credentials to gain access.
+Activities _MAY_ be published about content that has access restrictions. Clients _MUST NOT_ assume that they will be able to access every resource that is the object of an Activity, and _MUST NOT_ assume that it has been deleted if it is inaccessible. For example, the content might be protected by an authentication system that is denying access (an end user might be able to provide the right credentials to gain access) or there may be a temporary network outage preventing the content from being retrieved (the network will eventually be restored).
 
-Content may also change state from being available to being protected by access restrictions, or become available having previously been protected. There are no new activity classes for these situations, and the publisher _SHOULD_ issue the regular `Update` or `Delete` activities, if that is the behavior desired for harvesters.
+Content may also change from being available to being protected by access restrictions, or become available having previously been protected. There are no more specific activity classes for these situations, and the publisher _SHOULD_ issue the regular `Update` or `Delete` activities accordingly.
```

Comment:
Should it be "activity classes" or "activity types"?

Reply:
It should be consistent, but I seem to have used them interchangeably. Under the Activity section, I say that `type` is the "activity type", but for the `CollectionPage` (for example) I say it is a Class. Happy with the changes :+1:

Reply:
"classes" is only in the spec three times and replacing w/ "types" makes sense. "class" (singular) is only used when talking about a type definition, e.g. 

> The `type` _SHOULD_ be a class defined ...

Which I think make sense. Next push will make the change.